payments/paypal.go

```diff
@@ -232,3 +240,47 @@ func (p PaymentProvider) GetSubscription(subscriptionID string) (models.Member,
 
 	return m, nil
 }
+
+func (p *PaymentProvider) GetSubscription(subscriptionID string) (string, string, error) {
+	c, err := config.Load()
+	if err != nil {
+		fmt.Printf("error with config: %s", err)
```

Comment:
Why are using `Printf` here but in the line 254 you are using `logrus` `Errorf` instead?